src/app/shared/components/template/components/task-card/task-card.component.scss

```diff
@@ -69,6 +69,14 @@
     }
   }
 
+  &.background-secondary {
```

Comment:
nit(blocking)
It would be good to avoid potential confusion with things like global classes. I'd probably recommend we use data attributes which can match multiple strings, e.g. `[data-variant~='background-secondary']`